specs/core/0_beacon-chain.md

```diff
@@ -654,13 +658,22 @@ A valid block with slot `0` (the "genesis block") has the following values. Othe
 `STARTUP_STATE_ROOT` is the root of the initial state, computed by running the following code:
 
 ```python
-def on_startup(initial_validator_entries: List[Any], genesis_time: uint64, processed_pow_receipt_root: Hash32) -> BeaconState:
+def on_startup(current_validators: List[ValidatorRecord],
+               pre_fork_version: int,
+               post_fork_version: int,
+               fork_slot_number: int,
```

Comment:
I would say start with post fork version = pre fork version and fork slot number 2**64-1, and remove those two arguments. It's a startup function; at startup time you generally don't know exactly when you want to do your next fork.

Reply:
Done!